bin/test-naming-rules.js

```diff
@@ -1,4 +1,10 @@
 #! /usr/bin/env node
```

Comment:
These directives have to be first in the file or cause issues, so that's why the license comes after.